app/operations/candidates/remove_file.rb

```diff
@@ -1,40 +1,42 @@
 # frozen_string_literal: true
 
 class Candidates::RemoveFile < ApplicationOperation
-  include Dry::Monads[:result, :try]
+  include Dry::Monads[:result, :do]
 
   option :candidate, Types::Instance(Candidate)
   option :actor_account, Types::Instance(Account)
   option :file, Types::Instance(ActiveStorage::Attachment)
 
   def call
-    result = Try[ActiveRecord::RecordInvalid] do
```

Comment:
actually, when we work with files that are stored somewhere remote, we can consider keeping `try`, but list more possible exceptions :thinking:

Reply:
we may add rescue block if needed for the network errors